lib/cleverpush_flutter.dart

```diff
@@ -279,3 +295,91 @@ class CleverPush {
     return null;
   }
 }
+
+class CPNotification {
```

Comment:
Its already defined here:
https://github.com/cleverpush/cleverpush-flutter-sdk/blob/master/lib/src/notification.dart#L3

Reply:
Okay, let me remove it then.

Reply:
Removed.